synapse/rest/client/v2_alpha/account.py

```diff
@@ -520,27 +682,31 @@ def on_POST(self, request):
                 400, "Missing param three_pid_creds", Codes.MISSING_PARAM
             )
 
+        # We ignore the bind parameter as this endpoint no longer is able to bind threepids
```

Comment:
this comment looks a bit lost

Reply:
MSC2290 removes `bind` from this endpoint, so that's what this comment is talking about, but I think it could be worded better...

Alternatively I also don't think it serves much purpose, and will just remove it.